projects/optic/src/commands/capture/init.ts

```diff
@@ -27,34 +27,45 @@ export function captureConfigExample(
 ) {
   return `
 ${skipConfigUpdate ? `capture:\n  ${oasFile}:` : ''}
+    # 🔧 Runnable example with simple get requests. 
+    # Run with "optic capture ${oasFile} --update interactive" 
+    # You can change the server and the 'requests' section to experiment
     server:
-      # 🔧 Update this to the command to run your server.
-      # Optional: If omitted, Optic assumes the server is running or started elsewhere.
-      command: your-server-command
-      # 🔧 Update this url to where your server can be reached.
-      # Required: Can be overridden with '--server-override'.
-      url: http://localhost:8080
-      # 🔧 Update the readiness endpoint for Optic to validate before sending requests.
-      # Optional: If omitted, perform no readiness checking.
-      ready_endpoint: /
-
-    # 🔧 Specify either 'requests.run' or 'requests.send' to generate requests to hit your server
+      url: https://api.github.com
     requests:
-      # ℹ️ Requests should be sent to the Optic proxy, the address of which is injected into 'run.command's env as OPTIC_PROXY (or the value of 'run.proxy_variable').
-      run:
-        # 🔧 Specify a command that will generate traffic
-        command: your-test-command
-        # 🔧 OPTIC_PROXY is added to your command's env and contains the URL of an Optic's local reverse proxy. Your command should send its requests to this URL.
-        proxy_variable: OPTIC_PROXY
-      # 🔧 Or instead, craft requests for Optic send to your server
-      send:
-        - path: /
-          method: GET
-        - path: /users/create
-          method: POST
-          headers:
-            content-type: application/json;charset=UTF-8
-          data:
-            name: Hank
+      send: 
+      - path: /users/mojombo
+        method: GET
+      - path: /users/defunkt
+        method: GET
+      - path: /users/pjhyett/repos
+        method: GET
+      - path: /users/pjhyett/followers
+        method: GET
+      - path: /orgs/opticdev/repos
+        method: GET   
+      - path: /orgs/facebook/repos
+        method: GET   
+      - path: /orgs/opticdev/repos
+        method: GET
+      - path: /orgs/facebook/repos
+        method: GET
+      - path: /orgs/github/repos
+        method: GET   
```

Comment:
GET is the default, i think we should remove all the unnecessary "method" lines save for the first one where we can add a comment that it is the default method.